mmv1/products/dlp/InspectTemplate.yaml

```diff
@@ -127,6 +142,10 @@ properties:
                       description: |
                         Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names listed
                         at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                    - !ruby/object:Api::Type::String
```

Comment:
Why is this field indented this way and why are there three copies of it?

Reply:
`version` fields are a part of `infoType` block as specified in the [API](https://cloud.google.com/dlp/docs/reference/rest/v2/InfoType). The `version` field was missing in multiple `infoType` blocks. Hence, I've added this field inside four `infoType` blocks.